pkg/fanal/artifact/local/fs_test.go

```diff
@@ -2413,6 +2440,9 @@ func TestYAMLConfigScan(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			// Set fake UUID for consistent test results
+			uuid.SetFakeUUID(t, "3ff14136-e09f-4df9-80ea-%012d")
+
```

Comment:
IIUC we may not use fakeUUID for this test since this test does not check BlobID (BlobIDAnything is used).

Reply:
Thanks. I was not aware of that. Fixed in 811a3afd4f5e69db8c276f1994e0e58800c77cb7